themes/vocabulary_theme/templates/page-with-title.html

```diff
@@ -0,0 +1,23 @@
+{% extends "layout.html" %}
+
+{% block title %}{{ this.title }}{% endblock %}
+
```

Comment:
Why a new template? This page seems pretty simple.

Reply:
There's not an existing universal template without a table of contents, for a simple content page. This should cut down on creating custom one-off templates every time a standalone text page is added.

Reply:
Oh okay, the previous design had one so I had just assumed the redesign had one too. Thanks!